src/clj_kondo/impl/analyzer/namespace.clj

```diff
@@ -83,7 +83,9 @@
 
 (defn lint-alias-consistency [ctx ns-name alias]
   (let [config (:config ctx)]
-    (when-let [expected-alias (get-in config [:linters :consistent-alias :aliases ns-name])]
+    (when-let [expected-alias (or
+                               (get-in config [:linters :consistent-alias :aliases ns-name])
```

Comment:
One minor thing, for performance reasons you could get `(get-in config [:linters :consistent-alias :aliases])` only once and then re-use that result.

Reply:
Yes of course